cmake/onnxruntime_framework.cmake

```diff
@@ -77,6 +77,16 @@ endif()
 
 if (onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS)
   target_compile_definitions(onnxruntime_framework PRIVATE DEBUG_NODE_INPUTS_OUTPUTS)
+  if (onnxruntime_ENABLE_SQL)
+    find_package (SQLite3)
+    if (SQLITE3_FOUND)
+      include_directories(${SQLite3_INCLUDE_DIR})
+      target_link_libraries (onnxruntime_framework ${SQLite3_LIBRARY})
+    else()
+      message( FATAL_ERROR "Could not locate SQLite3 package." )
+    endif (SQLITE3_FOUND)
+    target_compile_definitions(onnxruntime_framework PRIVATE ENABLE_SQL)
```

Comment:
>ENABLE_SQL [](http://example.com/codeflow?start=61&length=10)

can this name be more specific and have the prefix "ORT"?

Reply:
I took the longer name you suggested above. But none of the other definitions throughout the code base have an ORT prefix. So I think it's better to not to add that.

Reply:
some of our preprocessor symbols don't have an ORT prefix, but I think they should because then naming conflicts with other code is less likely and there isn't another way to namespace them.

[https://google.github.io/styleguide/cppguide.html#Preprocessor_Macros](https://google.github.io/styleguide/cppguide.html#Preprocessor_Macros)
"Name macros with a project-specific prefix."

there are some that do - e.g. ORT_MINIMAL_BUILD

not blocking, just something to consider